### PR TITLE
Add minutes for meeting 21 Apr 2016

### PR DESCRIPTION
**Pull request by mhdawson**
@nodejs/api please review/comment

### CODE REVIEW

File: wg-meetings/2016-04-21.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+### Review of vm f-t-f
+Stefan walked us through plan
+https://docs.google.com/document/d/1No4LUnsbL4L_HJG06SrHcRSpmex2oYBvbLKFrKjD2RU/edit 
+
+IRC channel has been created: node-api in freenode.
+
+This doc has some initial notes/info about investigation
+by Stefan/Ian so far:
+
+https://docs.google.com/document/d/1No4LUnsbL4L_HJG06SrHcRSpmex2oYBvbLKFrKjD2RU/edit
```
Comment (trevnorris) [inline wg-meetings/2016-04-21.md]: Looks like you have this URL duplicated.

**Comment by orangemocha** [comment]
Rubber stamp LGTM

Minor suggestion: I would expand "f-t-f" to "face-to-face VM summit" because it might not be obvious to everyone what f-t-f stands for. Could also add a link to https://github.com/jasnell/vm-summit

**Comment by mhdawson** [comment]
@trevnorris  @orangemocha addressed comments.

File: wg-meetings/2016-04-21.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* Michael Dawson @mhdawson
+* Arunesh Chandra @aruneshc
+* Stefan Budeanu @stefanmb
+* Josh Gavant 
+* Jianchun
+* Alexis 
+
+## Round Table
+
+Went through stand up, mostly participation in vm f-to-f and then
```
Comment (orangemocha) [inline wg-meetings/2016-04-21.md]: Still says "f-to-f"

File: wg-meetings/2016-04-21.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+follow up actions from that.  Will try to take more specific notes next time.
+
+
+## Agenda items
+
+### Review of vm face-to-face VM summit (https://github.com/jasnell/vm-summit)
```
Comment (orangemocha) [inline wg-meetings/2016-04-21.md]: "vm" is repeated twice

File: wg-meetings/2016-04-21.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+## Actions for next meeting
+
+Michael to review README.md to see if we can improve
+positioning versus Nan wg.
+
+## Next meeting
+
+Some discussion about what deadlines we have from the vm
+f-to-f actions, in the end we agreed that we should
```
Comment (orangemocha) [inline wg-meetings/2016-04-21.md]: Still says "f-to-f"

File: wg-meetings/2016-04-21.md (hunk 2 above)
Comment (mhdawson) [inline wg-meetings/2016-04-21.md]: Sorry fixed title, but should have done more thourough search.

**Comment by mhdawson** [comment]
@orangemocha I think I got them all this time.

**Comment by orangemocha** [comment]
LGTM

**Comment by ianwjhalliday** [comment]
A link to the YouTube recording would be handy

File: wg-meetings/2016-04-21.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+* Open issues review
+
+## Present
+
+* Michael Dawson @mhdawson
+* Arunesh Chandra @aruneshc
+* Stefan Budeanu @stefanmb
+* Josh Gavant 
+* Jianchun
+* Alexis
```
Comment (ianwjhalliday) [inline wg-meetings/2016-04-21.md]: Fill in other names?

Josh Gavant @joshgav
Jianchun Xu @jianchun
Alexis Campailla @orangemocha

**Comment by mhdawson** [comment]
@ianwjhalliday addressed your comments. If I can get your ok I'll go ahead and land.

**Comment by ianwjhalliday** [comment]
👍

**Comment by mhdawson** [comment]
Landed as 65b55ee18c1daa9f4bac7da0ee0792e8841821f7

**Pull request closed without merging**